canonical-web-and-design/snapcraft.io
# Remove hardcoded omgubuntu blog posts

## Done
Removed hardcoded OMGUbuntu blog posts from Snap detail pages

## QA
- Go to https://snapcraft-io-3557.demos.haus/ncspot
- Check that there are no OMGUbuntu blog posts
- Go to https://snapcraft-io-3557.demos.haus/obs-studio
- Check that there are no OMGUbuntu blog posts
- Go to https://snapcraft-io-3557.demos.haus/standard-notes
- Check that there are no OMGUbuntu blog posts
- Go to https://snapcraft-io-3557.demos.haus/code
- Check that there are related blog posts

## Issue
Fixes #3485
